packages/query-core/src/index.ts

```diff
@@ -1,7 +1,7 @@
 /* istanbul ignore file */
 
 export { CancelledError } from './retryer'
-export { QueryCache } from './queryCache'
+export { QueryCache, QueryCacheNotifyEvent } from './queryCache'
```

Comment:
I think this needs to be:

```suggestion
export { QueryCache } from './queryCache'
export type { QueryCacheNotifyEvent } from './queryCache'
```